.travis.yml

```diff
@@ -43,14 +43,16 @@ jobs:
     script:
     - GOOS=darwin GOARCH=amd64 go build -ldflags="$LINKFLAGS"
     - tar czf ${PROJECT_NAME}-${TRAVIS_TAG}-x86_64-apple-darwin.tgz wash
+    - cp ${PROJECT_NAME}-${TRAVIS_TAG}-x86_64-apple-darwin.tgz ${PROJECT_NAME}-latest-x86_64-apple-darwin.tgz
```

Comment:
I'd drop `latest` from the name, I don't think it provides any useful information. Unless making it have the same number of dashes is useful...

Reply:
Makes sense. Updated again.